remo/voting/tasks.py

```diff
@@ -138,7 +141,7 @@ def create_rotm_poll():
                                     end__range=[start, end])
 
     if not now().date() > ROTM_NOMINATION_END_DATE or rotm_poll.exists():
```

Comment:
Also by not returning here, you are actually running code that's not supposed to be executed.